### PR TITLE
updated link to Docker Cloud and surrounding text

### PR DESCRIPTION
**Pull request by londoncalling**
### Proposed changes

Updated the link the Docker Cloud dashboard because the one on the docs site was giving me a 404 (see https://docs.docker.com/docker-cloud/getting-started/connect-infra/#/link-to-a-cloud-service-provider). 

I also re-worded the text in that paragraph to better match what I'm seeing on the Cloud login page and my dashboard.  (If you don't like any of this feel free to reject it :-) I've been going through the Cloud docs to learn about Cloud and follow along with the tutorials, so these copyedits are incidental to that. The docs look great really .. quite beautiful.)

### Please take a look

@sanscontext 

Signed-off-by: Victoria Bialas <victoria.bialas@docker.com>

### CODE REVIEW

**Comment by johndmulhausen** [comment]
Deploy preview processing.

Built with commit 746e820cbcc49b5db94274b39877984cad5f1dc1

https://app.netlify.com/sites/docker-docs/deploys/5825301fd6865d56cbeef3a9

**Comment by johndmulhausen** [comment]
Deploy preview processing.

Built with commit 746e820cbcc49b5db94274b39877984cad5f1dc1

https://app.netlify.com/sites/docker-docs/deploys/5825301f71e20a307ce0db20

File: docker-cloud/getting-started/connect-infra.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ To deploy Docker Cloud nodes, you first need to grant Docker Cloud access to you
 This could mean granting access to a cloud services provider such as AWS or Azure, or installing the Docker Cloud Agent on your own hosts. Once this is done, you can provision nodes directly from within Docker Cloud using the Web UI, CLI or API.
 
 ## Link to a cloud service provider
-To link your cloud provider accounts, first go to your [Docker Cloud account dashboard](https://cloud.docker.com/account/). (You can also click your username in the top right corner and choose **Account info**.)
+To link your cloud provider accounts, first go to your [Docker Cloud dashboard](https://cloud.docker.com/). If you are not already logged in, choose **Login with Docker ID** in the top right corner to do so. (On the dashboard, you'll find your username, account settings, and other options in the top right.)
```
Comment (sanscontext) [inline docker-cloud/getting-started/connect-infra.md]: I would actually prefer not to give login instructions here. If you put them here, you end up with them on every procedure. ;) 
I feel like this is signaled well enough from the UI that if they're interrupted by the sign in screen, they should be able to figure this out on their own.

**Comment by londoncalling** [comment]
@sanscontext cool! I deleted that second sentence re: logging on. As soon as you approve, I can merge, or you can :-)

**Pull request merged**